### ISSUE TITLE
perfdata should not end with c

### ISSUE THREAD

**Issue by tbc** [SamK/check_iftraffic_nrpe.py#30]
I had to patch as follows to get graphs to show up in Icinga Classic UI 1.11.5 (Backend 1.11.5). Am I missing something, or is this a bug?

```
diff --git a/check_iftraffic_nrpe.py b/check_iftraffic_nrpe.py
index f8a14d0..b7d281a 100755
--- a/check_iftraffic_nrpe.py
+++ b/check_iftraffic_nrpe.py
@@ -250,7 +250,7 @@ class NagiosService(object):
 
     def __str__(self):
         """Return the perfdata string"""
-        return '%(label)s=%(value).2fc;' \
+        return '%(label)s=%(value).2f;' \
                '%(warn_level)s;' \
                '%(crit_level)s;' \
                '%(min_level)s;' \
```

**Comment by donnyberger** [comment]
I guess this is a typo. We came along the same issue as you and had to patch this ourselves.

**Comment by SamK** [comment]
I there, thank you very much for your message,

I broke the thing at eaf473c5b74813ea0e8c39421ba9664d65917762 for some obscure reason.
I then tried to fix it at 3f3f0e39a6c25966b27c338026fb4ad78e55f4c8 but inserted the typo.

I will investigate the reason of this mess and provide a correction.

**Comment by SamK** [comment]
I double checked the Nagios API and the commit history and I figured out where it comes from.
The issue has been introduced in 3e273449a1d0f47c63fb347e3ece6da364ccf729 by issue #18.

The "c" thing means it's a *continous counter* (a number increasing over time).  However the output of check_iftraffic_nrpe.py is not a counter but a value of bytes per second (by default).

I've been confused with this "c" because the script was counting things at some point.

The [Nagios plugins guideline](https://nagios-plugins.org/doc/guidelines.html) does not specify a suitable unit for *bandwidth* (which is the actual proper type you are reading) 

The default unit (as stated in the README.md file) is bytes/seconds, configurable with `--unit`.

tldr; I am going to apply your patch.
